tests/feature_check/lineno.py

```diff
@@ -0,0 +1,6 @@
+import sys
```

Comment:
I can't see this file being used by `tests/run-tests.py`, I don't think it does anything having it here.

Reply:
I think I wasnt sure about how do we want to add new user facing tests to the project. I included it for completeness

Reply:
There are 2 ways to skip tests:

Reply:
Appreciate your comments and took note. I'm taking the test out for now and the PR looks in good shape.